src/main/java/com/machowina/controller/OwnerController.java

```diff
@@ -0,0 +1,40 @@
+package com.machowina.controller;
+
+import java.math.BigDecimal;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.machowina.model.ParkingZone;
+import com.machowina.service.ParkingZoneService;
+
+@RestController
+@RequestMapping("/owners")
+public class OwnerController {
+	
+	private final ParkingZoneService zoneService;
+	
+	@Autowired
+	public OwnerController(ParkingZoneService zoneService) {
+		super();
+		this.zoneService = zoneService;
+	}
+
+
+
+	@GetMapping("/zones/{zoneId}/income/yyyy-MM-dd/{incomeDate}")
```

Comment:
czy w endpoincie powinno być coś takiego `yyyy-MM-dd`? Co się stanie jak postanowimy rozszerzyć zbiór akceptowalnych formatów dat?

Reply:
Nie byłam pewna czy to tam umieszczać, ale czy w takim razie ta informacja powinna być tylko w dokumentacji?
Wydawało mi się że tak czytelniej widać jakiego formatu oczekujemy. Ale rzeczywiście jeśli docelowo zakładamy obsługiwanie wszystkich popularnych formatów to lepiej to wyrzucić.